src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1256,7 +1256,16 @@ private String readStdOut(Process p) throws IOException, InterruptedException {
     private String getContainerCommand() {
         StringBuilder command = new StringBuilder("docker run --rm");
         if (!skipDefaultPorts) {
-            command.append(" -p 9080:9080 -p 9443:9443");
+            int httpPortToUse, httpsPortToUse;
+            try {
+                httpPortToUse = findAvailablePort(9080, false);
+                httpsPortToUse = findAvailablePort(9443, false);
+            } catch (IOException x) {
+                httpPortToUse = 9080;
+                httpsPortToUse = 9443;
```

Comment:
We should also log the error/exception here.

Reply:
ok